packages/rest/src/__tests__/unit/rest.server/rest.server.unit.ts

```diff
@@ -171,10 +171,15 @@ describe('RestServer', () => {
         }
 
         get expressApp() {
+          assertExists(this._expressApp);
           return this._expressApp;
         }
       }
 
+      function assertExists<T>(val: T): asserts val is NonNullable<T> {
+        expect(val != null).to.be.true();
```

Comment:
I am concerned that when this assertion fail, it will print a message that's not very helpful. Have you considered using this?

```ts
expect.exists(val);
```

See http://shouldjs.github.io/#should-exist

```sh
$ node
> expect.exists(null)
Uncaught [Error [AssertionError]: expected null to exist] {
  stackStartFunction: [Function (anonymous)]
}
> expect.exists(undefined)
Uncaught [Error [AssertionError]: expected undefined to exist] {
  stackStartFunction: [Function (anonymous)]
}
```